### PR TITLE
Dast Automation(P1) - added new components & deprecated old ones 

### PR DESCRIPTION
**Pull request by SmitGala**

### CODE REVIEW

File: app/components/file-details/scan-actions/index.scss [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
+.scan-status-card {
   border: 1px solid var(--file-details-scan-actions-border-color);
-  background-color: var(--file-details-scan-actions-background-color);
-  box-sizing: border-box;
-
-  .scan-actions-header {
-    padding: 0.75em 1.5em;
-  }
+  background-color: white;
+  width: 240px;
```
Comment (SmitGala) [inline app/components/file-details/scan-actions/index.scss:4]: Changing This

File: app/components/file-details/index.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.file-details-root {
+  background: #faf9f9;
```
Comment (future-pirate-king) [inline app/components/file-details/index.scss:2]: use `--background-light`

File: app/components/file-details/index.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  margin: -0.5em;
+  padding: 0.5em;
+}
+
+.file-details-header {
+  padding: 0 4.285em;
```
Comment (future-pirate-king) [inline app/components/file-details/index.scss:8]: change container contraint by width

File: app/components/file-details/scan-actions/manual-scan/index.ts [hunk 4]
```diff
@@ -22,53 +13,13 @@ export interface FileDetailsScanActionsManualScanSignature {
...
-          'file-details/scan-actions/manual-scan/vpn-details' as const,
-      },
-    ];
-  }
 
   get manualScanStatusLabels() {
     return {
       [ENUMS.MANUAL.NONE]: this.intl.t('notStarted'),
       [ENUMS.MANUAL.REQUESTED]: this.intl.t('requested'),
       [ENUMS.MANUAL.ASSESSING]: this.intl.t('inProgress'),
```
Comment (future-pirate-king) [inline app/components/file-details/scan-actions/manual-scan/index.ts:21]: check this with design to refactor all to uppercase

File: tests/integration/components/file-details/scan-actions-test.js
Comment (future-pirate-king) [inline tests/integration/components/file-details/scan-actions-test.js:1]: move to acceptance test

File: tests/integration/components/file-details/scan-actions/api-scan-test.js
Comment (future-pirate-king) [inline tests/integration/components/file-details/scan-actions/api-scan-test.js:1]: move to acceptance test

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: tests/integration/components/file-details/scan-actions/dynamic-scan-test.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      await this.owner.lookup('service:organization').load();
+    });
+
+    test('it renders dynamic scan title & btn', async function (assert) {
+      this.server.get('/manualscans/:id', (schema, req) => {
+        return { id: req.params.id };
+      });
+
+      this.file.dynamicStatus = ENUMS.DYNAMIC_STATUS.NONE;
+      this.file.isDynamicDone = false;
```
Comment (future-pirate-king) [inline tests/integration/components/file-details/scan-actions/dynamic-scan-test.js:41]: add test for other scenarios too

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**